include/secp256k1.h

```diff
@@ -80,6 +80,26 @@ typedef struct {
     unsigned char data[64];
 } secp256k1_ecdsa_signature;
 
+/** Data structure that holds a sign-to-contract ("s2c") opening. Sign-to-contract
+ *  allows a signer to commit to some data as part of a signature. It can be used as
+ *  an Out-argument in certain signing functions.
+ *
+ *  This structure is not opaque, but it is strongly discouraged to read or write to
+ *  it directly.
+ *
+ *  The exact representation of data inside is implementation defined and not
+ *  guaranteed to be portable between different platforms or versions. It is however
+ *  guaranteed to be 73 bytes in size, and can be safely copied/moved.
```

Comment:
This guarantee is basically impossible to make for a data structure with multiple heterogeneous fields.

Reply:
"safely copied/moved" is OK, but the exact size is implementation dependent.

Reply:
(if it wasn't clear, due to alignment padding, which could basically be anything)

Reply:
yeah, not sure what's got into me

Reply:
fixed